crates/rspack_regex/Cargo.toml

```diff
@@ -20,4 +21,5 @@ swc_core = { workspace = true, features = [
   "__parser",
   "__utils",
   "ecma_preset_env",
+  "base",
```

Comment:
can you import the crate instead of base? I think I removed based everywhere ...

Reply:
Done